devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy

```diff
@@ -68,15 +56,10 @@ def updated() {
 }
 
 def configure() {
-	def cmds =  [
-			secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
-			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03)),
-			secure(zwave.switchBinaryV1.switchBinaryGet())
+	[
+        secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
```

Comment:
you sure you don't have a spaces->tabs function in your editor?

Reply:
I've recently switched to VS Code, and I was sure I've changed it. Anyway, there should be no more spacing issues from now.